### DISCUSSION TITLE
Ayuda con botón de pago de suscripción

### DISCUSSION THREAD

**Discussion by jjcampis** [mercadopago/sdk-js#196] [Q&A]
Hola estoy teniendo problemas con el botón de suscripciones me sirve, el tema es que al finalizar si el usuario no le da click a "volver al sitio" no puedo obtener el ID de approval dejo un enlace a una pregunta que hice en stack overflow de paso...

Lo que quiero es poder implementar la función onclose o bien redirigir en caso que sea aprobado el pago(sin que tenga que dar click)

https://es.stackoverflow.com/questions/612923/boton-suscripcion-mercadopago

**Comment by francineemilia** [comment]
@jjcampis ¿Puede decirnos qué documento del devsite de [mercadopago](https://developers.mercadopago.com/) utilizó para implementar el botón?

**Reply by jjcampis** [reply]
Hola @francineemilia es el boton que se genera cuando se crea una suscripcion, no de la parte developer, sino de usuario cree una suscripcion y te da un código para pegar en tu web el cual abre un modal.
![image](https://github.com/mercadopago/sdk-js/assets/4637091/7e966065-a953-4d30-ab1a-bfe98aa42454)
No hay documentacion alguna que diga como implementar ese callback, estoy desesperado.

**Comment by romerosilva-meli** [comment]
Hola @jjcampis, como estás?

esta discusión es exclusiva para asuntos relacionados con sdk js. Desafortunadamente tenemos conocimiento sobre este botón. Te pediré que [contactes el soporte](https://www.mercadopago.com/developers/es/support/center/tickets) para que ellos puedan ayudarte.
